lib/main.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:flutter/material.dart';
+
+import 'classes/top_stories.dart';
+import 'functions/all_functions.dart';
+import 'widgets/title_page.dart';
+import 'widgets/view_for_timelines.dart';
+import 'widgets/element_on_list.dart';
+
+const numberOfItems = 100;
+
+void main(List<String> args) {
+  runApp(const HNApp());
+}
+
+class HNApp extends StatelessWidget {
+  const HNApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      title: 'HN Reader',
+      home: HNMainScreen(),
+    );
+  }
+}
+
+class HNMainScreen extends StatefulWidget {
+  const HNMainScreen({super.key});
+
+  @override
+  State<HNMainScreen> createState() => _HNMainScreenState();
+}
+
+class _HNMainScreenState extends State<HNMainScreen> {
+  late Future<TopStories> newsFuture, questionsFuture, showFuture;
+  late List<List> news = <List<dynamic>>[];
+  final PageController _controller = PageController(initialPage: 0);
+  final List<String> tabs = <String>['Top Stories', 'Ask', 'Show'];
+  final List<String> typeOfStories = [
+    'topstories',
```

Comment:
I can see some of these string literals are re-used or very similar in a couple of places. Can you store them as constant values to re-use? That way you won't run into bugs like typos.